uber/custom_tags.py

```diff
@@ -15,6 +17,26 @@ def safe_string(text):
         return Markup(text)
 
 
+@JinjaEnv.jinja_filter
+def basename(s):
+    return os.path.basename(s) if s else ''
+
+
+@JinjaEnv.jinja_test(name='class')
+def is_class(value):
+    return inspect.isclass(value)
```

Comment:
I'm afraid I can't find what `inspect.isclass` actually DOES in Jinja2. Do you have a link to the documentation...? Their doc search isn't turning up anything, nor is Google.